vtex-apps/order-quote
# [ORDERQUOTE-7] Fix order quote to display all content from table when printing

#### What does this PR do? \*
Displays all the columns when printing order quote.

#### How to test it? \*
Test in workspace [annaorderquote](https://annaorderquote--sandboxusdev.myvtex.com/). Create order quote and click Print to see file.

_Before, printing in desktop view cut off the Total column_
<img width="631" alt="Screen Shot 2022-06-13 at 9 18 29 AM" src="https://user-images.githubusercontent.com/53097865/173362633-2f6a08c1-44a0-4a6a-952f-a3b000b185ca.png">

_After, printing displays all five columns_
<img width="584" alt="Screen Shot 2022-06-13 at 9 20 06 AM" src="https://user-images.githubusercontent.com/53097865/173362885-9a3707ae-a7f6-4854-8381-2aaf9ca82d20.png">
